eloquent/distribution.yaml

```diff
@@ -967,6 +967,17 @@ repositories:
       url: https://github.com/kobuki-base/kobuki_core.git
       version: devel
     status: maintained
+  kobuki_firmware:
+    doc:
+      type: git
+      url: https://github.com/kobuki-base/kobuki_firmware.git
```

Comment:
The package.xml lists the license as BSD for this repository, but it is distributing stm32flash (which is GPL, judging from https://sourceforge.net/p/stm32flash/code/ci/master/tree/main.c), and the kobuki firmware, which is under some other license (I'm not sure which one).  Questions:

1.  I believe we want to change the license field to GPL (at least).
1.  What's the license on the kobuki firmware?  Do we have permission to redistribute it?

Reply:
* There is no license on the kobuki firmware (we were naive engineers :)). In spirit also, we had no intention of blocking distribution of the binaries.
* Is it possible for the package to be of one license and internal parts thereof be of another license, so long as that license is installed in `share`?